samples/README.md

```diff
@@ -1855,6 +1856,23 @@ __Usage:__
 
 
 
+### Upload Without Auth
```

Comment:
### Upload without authenticating

Reply:
This is proper sentence case so it'll render correctly when added to docs. Also provides more detail as to what "auth" means.

Reply:
This is autogenerated by a bot and cannot be changed (without a ton of effort, like contacting the team that wrote the bot)

Reply:
I think I can change the word "auth" to "authenticating" by renaming the file the sample is in. Do you think i should make that change?

Reply:
I think it'd be best to make that change. "Upload without auth" seems like it could imply many things (upload without authenticating, upload without OAuth, etc)